core-bundle/src/Twig/Runtime/HighlightResult.php

```diff
@@ -15,7 +15,7 @@
 use Highlight\HighlightResult as BaseHighlightResult;
 
 /**
- * This class is a thin wrapper around @see \Highlight\HighlightResult that
+ * This class is a thin wrapper around @see HighlightResult that
```

Comment:
@leo Doesn’t this change the meaning and should be replaced with this instead?

```suggestion
 * This class is a thin wrapper around @see BaseHighlightResult that
```

Reply:
Of course. See #5633.